tests/search/indexed_tokens/indexed_tokens.rb

```diff
@@ -35,13 +36,15 @@ def test_lingustics_tokens
                          component(Component.new("com.yahoo.language.opennlp.OpenNlpLinguistics"))))
     start
     feed_doc("0", { :stext => "Hello world",
+                    :stext => "Hello cASed",
```

Comment:
Duplicate hash key. This will cause the test to fail when checking the `stext_tokens` summary field later on.

Reply:
Thanks!